sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -276,6 +277,27 @@ def transform(self, X):
         return sp.csr_matrix(X)
 
 
+class BinaryDecisionTreeClassifier(DecisionTreeClassifier):
```

Comment:
To avoid duplicate code maybe just inherit from `UntaggedBinaryDecisionTreeClassifier` and just override `_more_tags`.

Reply:
yep fair enough & done @NicolasHug